notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1136_M1M3_analysis_stability_tracking.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #36.            df_ims.index.get_indexer([pd.to_datetime(t_track_starts)+ pd.to_timedelta(10, unit="s")], method="nearest")
</pre><p> </p><p>This is just a question. Why you are summit 10s to get the index? Same question for the idxT1, why are you subtracting it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I do this because I want the plot to be centred on the Y-axis at 0 (a little lower I calculate <code>correctedVariablePlot</code>). To subtract the "continuous" I select in the dataframe information after 10 seconds from the start of the tracking and 10 seconds before the end to avoid entering slew data.</p><p> </p><p>I don't know if this clarification now makes sense.</p>

Reply:
<p>Tks!</p>